ts/store/reducers/wallet/transactions.ts

```diff
@@ -26,27 +31,34 @@ export const TRANSACTIONS_INITIAL_STATE: TransactionsState = {
 };
 
 // selectors
-export const getTransactions = (
-  state: GlobalState
-): IndexedById<WalletTransaction> => state.wallet.transactions.transactions;
+export const getTransactions = (state: GlobalState): IndexedById<Transaction> =>
+  state.wallet.transactions.transactions;
 export const getSelectedTransactionId = (state: GlobalState): Option<number> =>
   state.wallet.transactions.selectedTransactionId;
 
 export const latestTransactionsSelector = createSelector(
   getTransactions,
-  (transactions: IndexedById<WalletTransaction>) =>
+  (transactions: IndexedById<Transaction>) =>
     values(transactions)
-      .sort((a, b) => b.isoDatetime.localeCompare(a.isoDatetime))
+      .sort(
+        (a, b) =>
+          isNaN(getTransactionCreationDate(a) as any) ||
```

Comment:
Because `isNaN`'s signature is `(n: number) => boolean`, but it actually accepts a wider [range of types](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN)